docs/modules/plugins/partials/dynamic-variables-ui.adoc

```diff
@@ -57,3 +57,31 @@ ${source-code}
 ----
 Then `${source-code}` matches `.+Home.+`
 ----
+
+=== Browser window dimension
+
+The set of dynamic variables provides ability to access browser outer window height and outer window width.
```

Comment:
how can users understand what is `outer window`?

Reply:
You mean that we don't need to specify that there is outer window, don't you? Or there is wrong wording? Or we need to add additional information for user?

Reply:
I guess it would be enough just `window`